parquet-hadoop/src/test/java/org/apache/parquet/hadoop/rewrite/ParquetRewriterTest.java

```diff
@@ -107,7 +107,7 @@
 @RunWith(Parameterized.class)
 public class ParquetRewriterTest {
 
-  private final int numRecord = 100000;
+  private final int numRecord = 10000;
```

Comment:
This may result in a single page for each column chunk. Could you try following things:
- Make sure more than one page are produced by setting config like `parquet.page.value.count.threshold` and relevant page size check parameters at https://github.com/apache/parquet-java/blob/master/parquet-hadoop/README.md
- Parameterize `numRecord` (and page size check config) and by default use 10000?

Reply:
@wgtmac : thanks for your comment. Do you happen to know how I can run a single test in the repository after making my changes? I wanted to run only the `ParquetRewriterTest`, but have not figured out a good way to achieve that via mvn.

Reply:
```
cd ~/Projects/parquet-java   # replace with your project root directory

cd parquet-hadoop

mvn test -Dtest=org.apache.parquet.hadoop.rewrite.ParquetRewriterTest
```

Reply:
As far as I understand, the relevant config parameter is `parquet.page.row.count.limit` and I have changed that to be `num_records / 5`. Hope it looks good!